tests/flags.js

```diff
@@ -7,7 +7,7 @@ const spawn = require('cross-spawn');
 const fs = require('fs-extra');
 const os = require('os');
 const xml2js = require('xml2js');
-const byline = require('byline');
+const readline = require('readline');
```

Comment:
oh, byline was only used in tests?

Reply:
Does byline do anything that readline does not? Unless find in files has let me down this was the only use of byline.

Reply:
byline was in devDependencies so it must have been used for tests only.

It’s unclear to me what byline is providing over readline. But if the tests pass we’re all good, I’d say. We’re not doing anything complicated in the tests anyway.